easybuild/easyconfigs/k/KofamScan/KofamScan-1.3.0-gompi-2021b.eb

```
@@ -0,0 +1,57 @@
+easyblock = 'Binary'
+
+name = 'KofamScan'
+version = '1.3.0'
+
+homepage = 'https://www.genome.jp/tools/kofamkoala/'
+description = """KofamKOALA assigns K numbers to the user's sequence data by 
+HMMER/HMMSEARCH against KOfam (a customized HMM database of KEGG Orthologs 
+(KOs)).  K number assignments with scores above the predefined thresholds 
+for individual KOs are more reliable than other proposed assignments.  Such 
+high score assignments are highlighted with asterisks '*' in the output.  
+The K number assignments facilitate the interpretation of the annotation 
+results by linking the user's sequence data to the KEGG pathways and EC 
+numbers.  [code only: libraries and external binaries but no data]."""
+# software_license = 'MIT'
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+
+source_urls = [
+    'https://www.genome.jp/ftp/tools/kofam_scan/',  # add archives/kofam_scan-%(version)s/ if moved
```

```suggestion
    'https://www.genome.jp/ftp/tools/kofam_scan/',
    'https://www.genome.jp/ftp/tools/kofam_scan/archives/kofam_scan-%(version)s/',
```

should be possible to preempt this, EB will try to next url if it fails.

---

Thank you!